internal/providers/terraform/azure/testdata/lb_rule_test/lb_rule_test.tf

```diff
@@ -25,7 +25,6 @@ resource "azurerm_lb" "example" {
 }
 
 resource "azurerm_lb_rule" "rules" {
-  resource_group_name            = azurerm_resource_group.example.name
```

Comment:
Deprecated and removed https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/guides/3.0-upgrade-guide#resource-azurerm_lb_rule